### PR TITLE
:pencil: updating information

### LINKED ISSUE #17: Bot telegram info
+ [ ] description

+ [ ] about info

+ [x] picture

+ [ ] force @nothingelsematters to update it manually in telegram

### PR DESCRIPTION
**Pull request by nothingelsematters**
Closes #17 and ready for a master pr.

If @ShuffleZZZ wants to, he can:

+ [x] Update logo

+ [x] Edit [README](README.md) content (*or just leave a review for that*)

+ [x] Center the logo in [README](README.md)

### CODE REVIEW

**Comment by ShuffleZZZ** [comment]
Discuss

**Comment by nothingelsematters** [comment]
@ShuffleZZZ My thoughts:

1. It's better to bring codefactor back before merge (it'll be merged straight to master after)
2. It's better to make width of logo much less, because it is fucking giant on desktop

File: README.md [hunk 1]
```diff
@@ -57,5 +57,5 @@ In order to be able to launch the bot server locally you have to compile sources
 
 + `IS_LONGPOLL=true` for using long polling instead of web hooking (common usage case is local launch and debugging)
 
-<img src="info/logo.jpg" width=100%>
+[![Pull Party Bot](info/logo.jpg)](https://t.me/PullPartyBot)
```
Comment (ShuffleZZZ) [inline README.md]: We have to revert it back to html, it seems bad on the big screens.

Need to revert to `width=100%` or set `align=center`

File: README.md [hunk 2]
```diff
@@ -57,5 +84,6 @@ In order to be able to launch the bot server locally you have to compile sources
 
 + `IS_LONGPOLL=true` for using long polling instead of web hooking (common usage case is local launch and debugging)
 
-[![Pull Party Bot](info/logo-white-no_boarders.png)](https://t.me/PullPartyBot)
+
+<img src="info/logo-white-no_boarders.png" align="center">
```
Comment (ShuffleZZZ) [inline README.md:88]: It is still not centered 🤪 Needs fix

File: README.md [hunk 3]
```diff
@@ -11,22 +11,49 @@
...
+    > You can enter users with or without `@` symbol \
+    You can't create empty party \
+    `@admins` party is reserved and already created \
+    `@` prefix will be removed from party name \
+    `@` symbol is not allowed in party name \
+    You can't add single user to one group more than once
+
 + `/update partyName user2 user3...` - update an existing party
 
+    > Inherits `/create` method's rules \
```
Comment (nothingelsematters) [inline README.md:28]: > Inherits

That supposes some kind of inheritance, which is not quite a concept used here. I would propose "The rules being applied are the same as for `/create` method" or smth like that, u know?

File: README.md [hunk 4]
```diff
@@ -11,22 +11,49 @@
...
+
+
 + `/delete partyName1 partyName2...` - forget the parties like it never happened
 
+    > Availible only to admins \
+    You can't delete `@admins` party
+
++ `/clear` - forget all the parties in the chat
+
+    > Inherits `/delete` method's rules
```
Comment (nothingelsematters) [inline README.md:39]: The same applies here

File: README.md (hunk 2 above)
Comment (nothingelsematters) [reply README.md:88]: Probably, u can wrap this in a paragraph 🤪

**Comment by nothingelsematters** [comment]
Good job @ShuffleZZZ

**Comment by nothingelsematters** [comment]
omg, @ShuffleZZZ, u approved it. should i merge it and send a pr to master? xdd kkAmazed

**Comment by ShuffleZZZ** [comment]
@nothingelsematters It was so long that I do not remember if it is finished or not. You can merge, I guess 🤪

**Pull request merged**